instance.tf

```diff
@@ -0,0 +1,40 @@
+#Security group for this EC2 instance to allow ssh inbound on port 22
+resource "aws_security_group" "allow_ssh" {
+  name        = "ec2-instance"
+  description = "Allow SSH inbound traffic"
+  vpc_id      = aws_vpc.my-main-vpc.id
+
+  ingress {
+    description      = "ssh to VPC"
+    from_port        = 22
+    to_port          = 22
+    protocol         = "tcp"
+    cidr_blocks      = ["0.0.0.0/0"]
```

Comment:
It's not secure to open ssh to the entire world.
Instead, please limit it to the public IP of your ISP